README.md

```diff
@@ -1,2 +1,132 @@
-# mv-mintodolist-JSbp
-Repo based on mv-mintodolist for exercise: "Does my coding partner follow JS best practices?"
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="./src/imgs/todolist-logo.png" alt="logo" width="180"  height="auto" />
+  <br/>
+
+  <h3><b>Minimalist ToDo-List App</b></h3>
+  <p>This is a simple minimalist ToDo App which you can use to track your daily pending tasks.</p>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 [Minimalist ToDo List] <a name="about-project"></a>
+
+**[Minimalist ToDo-List]** is an App that you can use to track your daily tasks.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">Javascript</a></li>
+    <li><a href="https://webpack.js.org/">WebPack</a></li>
+  </ul>
+
+
+### Key Features <a name="key-features"></a>
+
+- **Extensive DOM Manipulation for Adding/Deleting tasks from the list.**
+- **Dinamically adds tasks at loading.**
+- **Mark a task as done.**
+- **Delete a task.**
+- **Delete all your completed tasks at once.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Minimalist ToDo-List - Live Demo](https://luigirazum.github.io/mv-mintodolist/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps:
+
+- [ ] Navigate to the button on the top right green "code" button.
+- [ ] In the drop down menu choose "Download Zip".
+- [ ] After extracting the files, you'll have my project on your local machine.
```

Comment:
The steps you specify are correct. I think in this case adding the commands to be run in the terminal like npm install and npm run start would help make your project usable. 👍 😄

Reply:
I already applied these changes ♻ too @cilfonegabriel, now you can see 👀 the `shell` instructions to have a local copy of the project up 👍 and running 👌.
Below 👇 there is an image that showcases the changes applied.
![image](https://user-images.githubusercontent.com/67355479/208761153-c8ada8bd-d94f-4938-a3b9-9aeb7c566c17.png)